ssa/tests/test_investigation_cas_humain_update.py

```diff
@@ -220,3 +220,47 @@ def test_update_adds_agent_and_structure_to_contacts(live_server, page, mocked_a
         mocked_authentification_user.agent.contact_set.get(),
         mocked_authentification_user.agent.structure.contact_set.get(),
     }
+
+
+def test_update_reference_souches_will_trigger_email(live_server, page, mailoutbox, mocked_authentification_user):
+    evenement = InvestigationCasHumainFactory(
+        not_bacterie=True, reference_souches="Test", etat=WithEtatMixin.Etat.EN_COURS
+    )
+    other_agent_contact = ContactAgentFactory()
+    evenement.contacts.add(other_agent_contact)
+    update_page = InvestigationCasHumainFormPage(page, live_server.url)
+    update_page.navigate_update_page(evenement)
+    update_page.reference_souches.fill("New value")
+    update_page.publish()
+
+    expect(update_page.page.get_by_text("L'évènement Investigation cas hummain a bien été modifié.")).to_be_visible()
```

Comment:
```suggestion
    expect(update_page.page.get_by_text("L'évènement Investigation cas humain a bien été modifié.")).to_be_visible()
```

Petite typo ici. C'est probablement ça qui pète les tests